src/diffusions.jl

```diff
@@ -69,11 +69,18 @@ function estimate_global_diffusion(::FixedDiffusion, integ)
     v, S = measurement.μ, measurement.Σ
     e = m_tmp.μ
     _S = _matmul!(Smat, S.R', S.R)
-    S_chol = cholesky!(_S)
     e .= v
-    ldiv!(S_chol, e)
+    if _S isa Kronecker.KroneckerProduct
+        @assert iszero(_S.A) || isone(_S.A)
```

Comment:
This is probably not actually the thing we want to check. Maybe rather make sure that the types are correct or so?